ignite/contrib/metrics/regression/fractional_bias.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import division
+
+import torch
+
+from ignite.exceptions import NotComputableError
+from ignite.metrics.metric import Metric
+
+
+class FractionalBias(Metric):
+    r"""
+    Calculates the Fractional Bias.
+
+    It has been proposed in `Performance Metrics (Error Measures) in Machine Learning Regression, Forecasting and
+    Prognostics: Properties and Typology`.
+
+    More details can be found in `https://arxiv.org/ftp/arxiv/papers/1809/1809.03006.pdf`.
+
+    :math:`\text{FB} = \frac{1}{n}\sum _j^n\frac{2 * (A_j - P_j)}{A_j + P_j}`
+
+    Where, :math:`A_j` is the ground truth and :math:`P_j` is the predicted value.
+
+    - `update` must receive output of the form `(y_pred, y)`.
+    - `y` and `y_pred` must be of same shape.
+
+    """
+    def reset(self):
+        self._sum_of_errors = 0.0
+        self._num_examples = 0
+
+    def update(self, output):
+        y_pred, y = output
+
+        denominator = y_pred + y.view_as(y_pred)
+
+        if (denominator == 0).any():
+            raise NotComputableError('The denominator has 0.')
```

Comment:
@vfdev-5 for this case, different from https://github.com/pytorch/ignite/pull/331, should we just leave a warning and return an `inf` or `nan` whichever the result is?  The comment in https://github.com/pytorch/ignite/pull/331#issuecomment-437721992 no longer applies, because here the `y_pred` is the value of model output, which is not under the control of user's hand. During training, `y_pred` could be any ridiculous number. Users might be interested in this metric (because when the model is good enough, this is well-defined), and doesn't care if occasionally this is ill-defined during training. In this case, raising `NotComputableError` might be annoying to the user because this stops their training.

Reply:
@zasdfgbnm I agree, thanks for pointing on this ! Let's use warnings.warn('The denominator is 0. Error becomes NaN') and set `_sum_of_errors` to np.nan.

Reply:
@vfdev-5 how about `inf`? Would it be better to leave it an `inf` if denominator is 0 but numerator is not?

Reply:
Mathematically correct would be plus or minus infinity. Maybe we can do similarly to torch:
```
import torch

torch.ones(2) / torch.eye(2)

tensor([[1., inf],
        [inf, 1.]])
```

Reply:
> I feel here we are a bit over-worried about divide by zero case. 

Yes, probably this is the best path to take :) Let's see others opinion. But I think you are right, we probably should just remove this `raise`.

Reply:
I agree with setting this as inf and removing the raise error.